test/CollapsibleSpec.js

```diff
@@ -31,6 +31,33 @@ describe('<Collapsible />', () => {
     assert(wrapper.find('[data-collapsible="accordion"]').length, 'with a data attribute');
   });
 
+  it('should work with null dynamic children', () => {
+    let wrapper = mount(
+      <Collapsible accordion>
+        {null}
+        <CollapsibleItem header='First'>
+          Lorem ipsum dolor sit amet.
+        </CollapsibleItem>
+      </Collapsible>
+    );
+
+    assert.strictEqual(wrapper.find('li').length, 1);
+  });
+
+  it('should pass the key props to its children', () => {
+    let collapsibleChildren = mount(
+      <Collapsible accordion>
+        <li key='testKey'>
```

Comment:
@isaiah: No, it shouldn't. Read above comments. It's not possible to get the key that way in Enzyme.